dashboard-server/queries/surveys/survey.test.js

```
@@ -92,23 +43,27 @@ describe('Survey queries', () => {
         shortcode: 123,
         userid: newUser.id,
         title: 'New User Title',
+        metadata: '{}',
+        survey_name: "test",
+        translation_conf: '{}',
       };
       const newSurvey = await Survey.create(survey);
+
       newSurvey.formid.should.equal('S8yR4');
       newSurvey.form.should.equal('{"form": "form detail"}');
-      newSurvey.shortcode.should.equal(123);
+      newSurvey.shortcode.should.equal('123');
```

Looks like this test was silently failing or not working.